File: Exemplos/App_eXperience_Java/Elgin_AppExperience_Java-font/app/src/main/java/com/elgin/elginexperience/shipay/responses/AuthenticationResponse.java
package com.elgin.elginexperience.shipay.responses;

import com.google.gson.annotations.SerializedName;

public class AuthenticationResponse {

    @SerializedName("access_token")
    public String access_token;

    @SerializedName("refresh_token")
    public String refresh_token;
}

File: Exemplos/App_eXperience_Kotlin/Elgin_AppExperience_Kotlin-font/E1_KOTLIN_R/app/src/main/java/com/example/e1_kotlin_r/shipay/responses/AuthenticationResponse.kt
package com.example.e1_elgin_kotlin.shipay.responses

import com.google.gson.annotations.SerializedName

data class AuthenticationResponse (
    @SerializedName("access_token")
    var access_token: String,

    @SerializedName("refresh_token")
    var refresh_token: String
)
